packages/dtslint/README.md

```diff
@@ -199,8 +193,6 @@ npm run watch
 
 Use `pnpm test` to run all tests.
 
-To run a single test: `node node_modules/tslint/bin/tslint --rules-dir dist/rules --test test/expect`.
```

Comment:
@jakebailey I remember you thought that `cd types/x; eslint .` would work now. Is that the case? Should we recommend that?

Reply:
It unfortunately would not. Because we are using a Program to list the files in a package, we actually _aren't_ linting a whole bunch of files in the repo. It turns out that there's many declarations that aren't in a project. For example, see: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/freedom/freedom-core-env.d.ts

I don't think this PR should try and deal with it, but it is something to solve.